pkg/plugins/plugins.go

```diff
@@ -119,6 +119,9 @@ type JSONData struct {
 
 	// App Service Auth Registration
 	IAM *pfs.IAM `json:"iam,omitempty"`
+
+	// API Version: Temporary field while plugins don't expose a OpenAPI schema
```

Comment:
Just curious, what would this start to look like long term - The API version won't be relevant and instead the schema represented by the API version will exist here instead?

Reply:
I believe so. Either some new metadata file (e.g. `plugin.schema.json`) or a new `schema` field that describe the plugin API (`config` and `query`, as we had). That should be the source of truth for this `APIVersion` value.

Reply:
Long term, the source of truth should be from the apiservers -- not plugin.json.  We need *something*  (maybe plugin.json) that will populate:
<img width="570" alt="image" src="https://github.com/grafana/grafana/assets/705951/c1cefcf8-bf8f-4981-adf9-9448fe57d1e0">

Given that we also need to define the schema for each apiversion, it will likely be something outside of plugin.json